### ISSUE TITLE
BlockVariationTransforms: Replace implementation with ToggleGroupControl

### ISSUE THREAD

**Issue by mirka** [WordPress/gutenberg#43415]
These buttons (`.block-editor-block-variation-transforms`) should be replaced with the new icon button variant of ToggleGroupControl.

<img width="134" alt="BlockVariationTransforms component in the Group block inspector" src="https://user-images.githubusercontent.com/555336/185595480-209b53ab-83ab-4398-9638-0b956d66bda1.png">

This will improve consistency, reduce code duplication, and add visual/semantic indications that the options are mutually exclusive.

**Comment by daledupreez** [comment]
I am taking a look at this issue.

**Comment by andrewserong** [comment]
There's also https://github.com/WordPress/gutenberg/pull/45654 that's looking at implementing this
